src/service/performance-impl.js

```diff
@@ -60,10 +60,8 @@ function getElementType(node) {
   if (node == null) {
     return ELEMENT_TYPE.other;
   }
-  const {tagName} = getOutermostAmpElement(node);
-  if (tagName == null) {
```

Comment:
I find that a bit confusing because of the `!`. It will technically work, but the fact that I have to reason about "what if it was nullish", "what does optional chain return if nullish", and "what's the not of the nullish optional chain" makes it all harder to reason about.